config.go

```diff
@@ -10,6 +10,7 @@ import (
 
 type config struct {
 	Domain       string       `json:"domain"`
+	Docs         string       `json:"docs"`
```

Comment:
I think we should include the word domain in this config value, e.g. `docsDomain`, so that it is clear that this value is a domain.
```suggestion
	DocsDomain   string       `json:"docsDomain"`
```

Reply:
done